pom.xml

```diff
@@ -622,18 +639,34 @@
 
     <profiles>
         <profile>
-            <id>hive-v2.3</id>
+            <id>hadoop2</id>
+            <activation>
+                <property>
+                    <name>hadoop</name>
+                    <value>v2</value>
+                </property>
+            </activation>
             <properties>
-                <hive.version>2.3.7</hive.version>
+                <guava.version>14.0.1</guava.version>
+                <hive.version>2.1.1</hive.version>
+                <hadoop.version>2.9.2</hadoop.version>
+                <terminal.spark.version>3.1.2</terminal.spark.version>
+                <terminal.spark.major.version>3.1</terminal.spark.major.version>
             </properties>
         </profile>
         <profile>
-            <id>hive-v2.1</id>
+            <id>hadoop3</id>
```

Comment:
In hadoop3, whether to consider using hadoop-client-api and hadoop-client-runtime, they exclude third-party dependencies, we can refer to spark

Reply:
his will make our pom file difficult to be compatible with Hadoop 2/3 in the simplest way, so I will still use a unified dependency approach in this PR.